.gitignore

```diff
@@ -1,3 +1,10 @@
+# Ignore every folders in keyboards exepct few that i use
+
+/keyboards/*
+!/keyboards/keebio/
+
+
+
```

Comment:
```suggestion
```

Reply:
Sorry my mistake.. I editted wrong project folder. Thank you for fixin it